cpp/src/arrow/filesystem/CMakeLists.txt

```diff
@@ -121,6 +121,24 @@ if(ARROW_S3)
       target_link_libraries(arrow-filesystem-s3fs-benchmark PRIVATE parquet_shared)
     endif()
   endif()
+
+  if(ARROW_S3_MODULE)
+    add_arrow_test(s3fs_module_test
+                   SOURCES
+                   s3fs_module_test.cc
+                   s3_test_util.cc
+                   EXTRA_LABELS
+                   filesystem
+                   DEFINITIONS
+                   ARROW_S3_LIBPATH="$<TARGET_FILE:arrow_s3fs>"
+                   EXTRA_LINK_LIBS
+                   Boost::filesystem
+                   Boost::system)
```

Comment:
@assignUser does the auto-formatter for CMake has the ability to indent these?

```cmake
                   SOURCES
                     s3fs_module_test.cc
                     s3_test_util.cc
                   EXTRA_LABELS
                     filesystem
                   DEFINITIONS
                     ARROW_S3_LIBPATH="$<TARGET_FILE:arrow_s3fs>"
                   EXTRA_LINK_LIBS
                     Boost::filesystem
                     Boost::system)
```

Reply:
Sorry, missed that ping. But yes, that is possible  Ben says, I have done something similar for velox custom functions. But currently I am a little concerned that cmake-format seems to be unmaintained https://github.com/cheshirekow/cmake_format/issues/340